crates/biome_js_analyze/src/analyzers/a11y/use_anchor_content.rs

```diff
@@ -117,8 +120,32 @@ impl Rule for UseAnchorContent {
             markup! {
                 "All links on a page should have content that is accessible to screen readers."
             }
+        ).note(
+            markup! {
+                "Accessible content refers to digital content that is designed and structured in a way that makes it easy for people with disabilities to access, understand, and interact with using assistive technologies. Follow the links for more information,
+                "<Hyperlink href="https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-in-context">"WCAG 2.4.4"</Hyperlink>"
+                "<Hyperlink href="https://www.w3.org/WAI/WCAG21/Understanding/name-role-value">"WCAG 4.1.2"</Hyperlink>""
```

Comment:
It seems that this text goes on a new line with a lot of space. Is it intended? 
![Screenshot 2023-10-06 at 08 51 32](https://github.com/biomejs/biome/assets/602478/85271e7b-59d6-41d4-932b-f58045566313)

Reply:
Oh, forgot the linebreaks.